### PR TITLE
docs(website): document decision to use Playwright

### PR DESCRIPTION
**Pull request by ace-n**
Related to #175 + builds on top of #363

/cc @grayside as FYI

### CODE REVIEW

**Review by pattishin — APPROVED** [review]
LGTM

File: docs/decisions/2022-04-website-testing-framework.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# End-to-End Website Testing Framework
+
+* **Status:** approved
+* **Last Updated:** 2022-04
+* **Builds on:** [Website Testing Philosophy](2022-04-website-testing-philosophy.md)
+* **Objective:** [description or link to contextual issue]
+
+## Context & Problem Statement
+
+There are several web-testing frameworks we could choose from. We need to determine which is best for the Emblem website's specific needs.
```
Comment (grayside) [inline docs/decisions/2022-04-website-testing-framework.md:10]: Why not a simpler HTTP response testing? What's the motivation for needing rendered pages?

Comment (ace-n) [reply docs/decisions/2022-04-website-testing-framework.md:10]: Lit is a clientside framework, and requires a browser.

**Review by engelke — APPROVED** [review]
LGTM

**Pull request merged**